sage-ios/SAGE/SAGE/LoginView.swift

```diff
@@ -0,0 +1,236 @@
+//
+//  LoginView.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/7/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+
+import UIKit
+
+class LoginView: UIView {
+    
+    var loginEmailField: UITextField?
+    var loginPasswordField: UITextField?
+    var signUpLink: UIButton?
+    var sageLabel: UILabel?
+    var containerView: UIView?
+    var firstDivider: UIView?
+    var secondDivider: UIView?
+    var loginButton: UIButton?
+    var loginButtonText: UILabel?
+    
+    var movedUp: Bool = false
+    
+    required override init(frame: CGRect) {
+        let screenRect = UIScreen.mainScreen().bounds;
+        let screenWidth = screenRect.size.width;
+        let screenHeight = screenRect.size.height;
+        let newFrame = CGRectMake(0, 0, screenWidth, screenHeight)
+        super.init(frame: newFrame)
```

Comment:
the frame that's passed in is 0,0,0,0 and I didn't want to pass in an actual frame when i created the view controller

is there a better way to do this??